cypress/integration/home_page_spec.js

```diff
@@ -0,0 +1,50 @@
+import { aliasQuery, hasOperationName } from '../utils/graphql-test-utils';
+
+describe('Not Fur-gotten Home Page User Flow', () => {
```

Comment:
Excellent job with stubbing this data, and our collective working of finding each individual piece needed to make this happen. Go, team!